maven/bnd-export-plugin/src/test/resources/integration-test/export-test/pom.xml

```diff
@@ -0,0 +1,30 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>biz.aQute.bnd-test</groupId>
+    <artifactId>export-test</artifactId>
+    <version>3.2.0-SNAPSHOT</version>
```

Comment:
The test pom should just be some simple version like 0.0.1. See https://github.com/bndtools/bnd/blob/master/maven/bnd-maven-plugin/src/test/resources/integration-test/test/pom.xml#L6